cmd/kube-scheduler/app/options/options.go

```diff
@@ -99,6 +93,18 @@ func NewOptions() (*Options, error) {
 	return o, nil
 }
 
+// Complete completes the remaining instantiation of the options obj.
+// In particular, it injects the latest internal versioned ComponentConfig.
+func (o *Options) Complete() error {
+	cfg, err := latest.Default()
+	if err != nil {
+		return err
+	}
+
+	o.ComponentConfig = *cfg
```

Comment:
Did we get rid of all the flags that affect the component config?

Reply:
> Did we get rid of all the flags that affect the component config?

We shouldn't. Please check the latest commit which reset the deprecated settings which have been parsed by flag.